openprescribing/frontend/tests/functional/mock_api_server.py

```diff
@@ -98,6 +98,7 @@ def do_GET(self):
         else:
             logger.error("Unhandled path %s" % self.path)
             self.send_response(requests.codes.not_found)
+            data = str(data)
```

Comment:
Some functional tests request `/api/1.0/spending/` from this mock api server, which isn't implemented, so it falls back to attempting to return the default `{}`, which being a `dict` doesn't support `encode('utf-8')`. If we convert to string here, our test doesn't crash.